docs/cas-server-documentation/configuration/Configuration-Properties.md

```diff
@@ -4388,6 +4389,37 @@ To learn more about this topic, [please review this guide](../ticketing/Ehcache-
 
 Signing & encryption settings for this registry are available [here](Configuration-Properties-Common.html#signing--encryption) under the configuration key `cas.ticket.registry.ehcache`.
 
+### Ehcache 3 Ticket Registry
+
+To learn more about this topic, [please review this guide](../ticketing/Ehcache3-Ticket-Registry.html).
```

Comment:
I don't mind this, but I would prefer to keep the documentation for ehcache all in one page. Makes it easier to update and/or remove deprecated content in the future.